src/Listeners/AppendTermSeoFieldsListener.php

```diff
@@ -27,9 +27,10 @@ public function handle(TermBlueprintFound $event)
             $contents = $bp->contents();
 
             $on_page_bp = OnPageSeoBlueprint::requestBlueprint();
-            $on_page_fields = $on_page_bp->contents()['sections']['main'];
+            $on_page_key = (array_key_exists('tabs', $on_page_bp->contents() ? 'tabs' : 'sections');
```

Comment:
missing `)` after `(array_key_exists('tabs', $on_page_bp->contents()`

is now;
`$on_page_key = (array_key_exists('tabs', $on_page_bp->contents() ? 'tabs' : 'sections');`
should be;
`$on_page_key = (array_key_exists('tabs', $on_page_bp->contents()) ? 'tabs' : 'sections');`